### ISSUE TITLE
LineChart alternate syntax

### ISSUE THREAD

**Issue by lukewis** [mermaid-js/mermaid#5087]
### Proposal

I noticed when creating a line chart that the user MUST specify a value for every increment along the x-axis.  In some cases, the user may want to supply only SOME of those values.  For example, start at (0,0) and draw a line to (50,100).  Calculating the slope of that line (and thus) the value at each increment is pretty arduous.  It might be nice to support an alternate syntax (in addition to the existing one)

### Example

```
# Existing
line [1,5,10,20]  # These are y-axis values for EACH x-axis increment

# Proposed
line [(1,1), (50,100)] # These specify actual coordinates so that the user does not have to specify ALL increments.
```

**Comment by sidharthv96** [comment]
CC @subhash-halder

**Comment by subhash-halder** [comment]
Yes, the syntax is very open so it will be powerful but may confuse many, it would be better if we could come up with a better use case for why we need it.
It would be better if we take more feedback from users, @sidharthv96 can we post it in the Slack group and gather feedback.
